modules/node/persistence/cms_node_storage_sql.e

```diff
@@ -561,7 +569,7 @@ feature {NONE} -- Queries
 			-- Physical deletion with free metadata.		
 
 	sql_restore_node: STRING = "UPDATE nodes SET changed=:changed, status =:status WHERE nid=:nid"
-			-- Restore node to  {CMS_NODE_API}.not_publised.
+			-- Restore node to  {CMS_NODE_API}.published
```

Comment:
you could rename it

sql_update_node_status
as it could be used to restore or even trash a node.